src/MpesaDaraja/Http/C2B/RegisterUrls.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Date 15/03/2023
+ *
+ * @author   kelvin mukotso <kelvinmukotso@gmail.com>
+ */
+
+namespace Ssiva\MpesaDaraja\Http\C2B;
+
+use Ssiva\MpesaDaraja\Http\AbstractDarajaQuery;
+
+class RegisterUrls extends AbstractDarajaQuery
+{
+    protected string $endpoint = 'mpesa/c2b/v1/registerurl';
+    /*
+    * Register Validation and confirmation Urls for
+    */
+    public function submitRequest(array $params = [], string $app = 'default')
+    {
+
+        $initiator = configStore()->get('mpesa.c2b.initiator_name');
```

Comment:
Only 4 $configParams are needed for src/MpesaDaraja/Http/C2B/RegisterUrls.php.

        $shortCode = configStore()->get('mpesa.c2b.short_code');
        $responseType = configStore()->get('mpesa.c2b.response_type');
        $confirmationUrl = configStore()->get('mpesa.c2b.confirmation_url');
        $validationUrl = configStore()->get('mpesa.c2b.validation_url');